virttest/storage.py

```diff
@@ -69,8 +69,7 @@ def file_exists(params, filename_path):
 
     :return: True if image file exists else False
     """
-    gluster_image = params.get("gluster_brick")
-    if gluster_image:
+    if params.get("enable_gluster") == "yes":
```

Comment:
could be `params.get_boolean("enable_gluster")`

Reply:
this is better, done.